champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/ranger/axe/Agility.java

```diff
@@ -121,7 +121,7 @@ public void onUpdate() {
     @Override
     public void activate(Player player, int level) {
         if (!active.contains(player.getUniqueId())) {
-            player.addPotionEffect(new PotionEffect(PotionEffectType.SPEED, (int) ((baseDuration + level) * 20), 1));
+            player.addPotionEffect(new PotionEffect(PotionEffectType.SPEED, (int) ((baseDuration + level) * 20), 3));
```

Comment:
That's actually speed 4

Agility currently has Speed 2 but the description is wrong